lib/calendar_assistant/cli/repl.rb

```diff
@@ -0,0 +1,70 @@
+require 'readline'
+
+class CalendarAssistant
+  module CLI
+    class Repl
+      class History
+        HISTORY_FILE = '~/.thor_interactive_history'
+        attr_reader :history_file
+
+        def initialize(readline_history = Readline::HISTORY)
+          @readline_history = readline_history
+          _history_file = ENV.fetch('THOR_HISTORY_FILE', History::HISTORY_FILE)
```

Comment:
Should we just make this a proper config parameter for the `interactive` command?